plugin/src/main/java/net/dzikoysk/funnyguilds/config/sections/HeartConfiguration.java

```diff
@@ -31,6 +31,10 @@ public class HeartConfiguration extends OkaeriConfig {
     @Comment("UWAGA: Jeśli jako serca gildii chcesz użyć bloku, który spada pod wpływem grawitacji - upewnij się, ze będzie on stał na jakimś innym bloku!")
     @Comment("Jeśli pojawi sie w powietrzu - spadnie i plugin nie będzie odczytywał go poprawnie!")
     public String createType = "ender_crystal";
+
+    @Comment("")
+    @Comment("Czy gracz powinien byc teleportowany do serca gildii przy jej zakładaniu")
```

Comment:
Dear @oskarscot,

I would like to bring to your attention a spelling mistake in the description regarding guild creation in the game, which can be found on the project page. The sentence "Czy gracz powinien byc teleportowany do serca gildii przy jej zakładaniu" lacks a proper Polish diacritic mark in the word "be" and is not correctly punctuated with a period at the end.

According to the rules of the Polish language, the word "be" should be written with a diacritic mark "ć". The lack of this mark renders the sentence grammatically incorrect and incomprehensible to the reader. Furthermore, the sentence should be properly punctuated with a period to maintain the correct sentence structure.

As individuals creating a project on GitHub, we should strive to maintain linguistic accuracy and proper formatting in our descriptions to ensure clarity and professionalism. Linguistic accuracy is especially important in a business context as it affects the company's image and reputation.

Therefore, I encourage all users to adhere to the rules of proper Polish grammar in project descriptions on GitHub and to ensure the correctness of the language form to guarantee professionalism and readability of our projects.

Best regards,
crejk

Reply:
Dear @crejk,

Thank you for bringing this issue to our attention. We have reviewed the project page and corrected the spelling mistake and punctuation error in the sentence you mentioned regarding guild creation. We apologize for any inconvenience or confusion this may have caused.

We appreciate your attention to detail and dedication to maintaining linguistic accuracy in project descriptions on GitHub. It is important to us that our projects are presented in a clear, professional, and readable manner.

Once again, thank you for your contribution and for holding us accountable. We look forward to continuing to improve our projects on GitHub with the help of our community.

Best regards,
@oskarscot

Reply:
Dear @oskarscot,

Thank you for your prompt response and attention to the matter we brought up in our previous message. We appreciate your efforts in correcting the spelling mistake and punctuation error in the project page regarding guild creation.

We understand that maintaining clear and professional communication in project descriptions is crucial for effective collaboration and ensuring that the intended message is conveyed accurately. Therefore, we applaud your commitment to upholding linguistic accuracy in your project descriptions on GitHub.

We also appreciate your acknowledgment of our contribution and dedication to maintaining linguistic accuracy. We believe that open and transparent communication is vital in fostering a productive and collaborative environment, and we are pleased to see that you share this belief.

Once again, thank you for your prompt and professional response. We look forward to continuing to work with you and contributing to the improvement of your projects on GitHub.

Best regards,
crejk

Reply:
xD